lib/wasix/src/lib.rs

```diff
@@ -549,13 +549,16 @@ fn wasix_exports_32(mut store: &mut impl AsStoreMut, env: &FunctionEnv<WasiEnv>)
         "proc_fork" => Function::new_typed_with_env(&mut store, env, proc_fork::<Memory32>),
         "proc_join" => Function::new_typed_with_env(&mut store, env, proc_join::<Memory32>),
         "proc_signal" => Function::new_typed_with_env(&mut store, env, proc_signal::<Memory32>),
+        "proc_signals_count_get" => Function::new_typed_with_env(&mut store, env, proc_signals_count_get::<Memory32>),
```

Comment:
The get seems redundant. IMO we should name this `proc_signals_count`

Reply:
Count is already a getter. In WASI they have `args_sizes_get`. We can have `proc_signals_sizes_get` which is consistent. But not `count_get` those are two verbs. We should either `count`, or `get`.